types/leaflet/index.d.ts

```diff
@@ -1534,6 +1535,9 @@ export interface MarkerOptions extends InteractiveLayerOptions {
     opacity?: number;
     riseOnHover?: boolean;
     riseOffset?: number;
+    autoPan?: boolean;
```

Comment:
It might be nice to add tests covering the new properties.

Reply:
Done